lego-mindstorms-pieces.py

```diff
@@ -339,67 +365,132 @@ def order(shop=None, browser=None, lego_set=None, order_list=None, username=None
             counter=counter,
             total_elements=total_elements), end='')
 
-        element_field = wait.until(EC.element_to_be_clickable(
-            (By.ID, 'element-filter')))
+        original_part_no = int(part_no)
 
-        element_field.clear()
-        element_field.send_keys(part_no)
-        element_field.send_keys(Keys.RETURN)
-        sleep(.3)  # seconds
+        # never add the same part twice,
+        # otherwise the quantity will be set to the previous part
+        if part_no in added_part.keys():
+            duplicate_part_counter += 1
+            print("IGNORE: Already added !".format(pn=part_no))
```

Comment:
French spacing again ! (Got you!) :smile:

Reply:
Man, "you really got me" :musical_note: